screenpy/resolutions/is_empty.py

```diff
@@ -15,5 +16,9 @@ class IsEmpty(BaseResolution):
         the_actor.should(See.the(List.of_all(VIDEO_FRAMES), IsEmpty()))
     """
 
+    matcher: _IsEmpty
     line = "an empty collection"
     matcher_function = empty
+
```

Comment:
I had to bypass pylint's `useless-super-delegation` rule to get this to commit.  We should talk about how to handle pylint rules.  (maybe use pylintrc file?)